examples/cfd/aero_graph_net/utils.py

```diff
@@ -14,7 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Any, Mapping, Optional
+
 import torch
+from torch import Tensor
+
+from dgl import DGLGraph
+
+
+class RRMSELoss(torch.nn.Module):
```

Comment:
Can this be used instead? https://github.com/NVIDIA/modulus/blob/main/modulus/metrics/general/mse.py#L47

Reply:
Hmm, but that one is a different function, right? That is, Ahmed body (and subsequently my code), uses RRMSE (**Relative** Root Mean Squared Error) loss, while the function you mentioned is a simple RMSE.
Now, I don't know why RRMSE was used in the original code, @ktangsali any comments?

Reply:
Great question - yeah I agree, the rmse from metrics might not be useful here. As to Alexey's question why RRMSE was used, I believe @mnabian might know best as he trained the ahmed body example initially.